app/views/home/team.html.erb

```diff
@@ -0,0 +1,44 @@
+<h1>Meet the team</h1>

```

Comment:
Can you add the nav bar to the top of this page? (every page really)

Reply:
Done.